docs/framework/xaml-services/xml-space-handling-in-xaml.md

```diff
@@ -26,14 +26,14 @@ The `xml:space` attribute is an XML-defined attribute that declares the signific
 ## Remarks  
  The definition for the `xml:space` attribute in XAML including its two possible values is derived from `xml:space` as defined as a "special attribute" by W3C specifications for XML.  
   
- The default value of the `xml:space` attribute is the literal value `"default"`. For the value `"default"`, or if `xml:space` is not indicated at all, the behavior of significant whitespace parsing is the default handling, as defined in the topic [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
+ The default value of the `xml:space` attribute is the literal value `"default"`. For the value `"default"`, or if `xml:space` is not indicated at all, the behavior of significant white space parsing is the default handling, as defined in the topic [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
   
- To preserve whitespace within object element content, specify `xml:space="preserve"` on that object element.  
+ To preserve white space within object element content, specify `xml:space="preserve"` on that object element.  
   
  Under most interpretations, the `xml:space` attribute effects and the value of the attribute are scoped to child elements.  
   
- For a complete discussion of whitespace processing in XAML, see [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
+ For a complete discussion of white space processing in XAML, see [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
```

Comment:
white space processing --> white-space processing
in the title: White space Processing --> White-space processing